### ISSUE TITLE
Test new moodle plugin block_scheduledcontent

### ISSUE THREAD

**Issue by rschrenk** [center-for-learning-management/moodle-block_scheduledcontent#1]
Dear Julia,

I have just created a new moodle plugin called block_scheduledcontent, which serves contents based on scheduled times. This makes it possible for us to make certain contents visible on planned time frames -> think of 1st of April, Advent calendar and so on.

Please play around a little and test the functionality.

Still missing is the possibility to delete certain schedules (for now you can only delete the whole block). But this first version will suit our plans for April.

Greetings
Robert

**Comment by lassnj** [comment]
Warnings on t1: 

```
Did you remember to call setType() for 'courseid'? Defaulting to PARAM_RAW cleaning.
line 1395 of /lib/formslib.php: call to debugging()
line 322 of /lib/formslib.php: call to moodleform->detectMissingSetType()
line 228 of /lib/formslib.php: call to moodleform->_process_submission()
line 68 of /blocks/scheduledcontent/schedules.php: call to moodleform->__construct()
Did you remember to call setType() for 'elements'? Defaulting to PARAM_RAW cleaning.
line 1395 of /lib/formslib.php: call to debugging()
line 322 of /lib/formslib.php: call to moodleform->detectMissingSetType()
line 228 of /lib/formslib.php: call to moodleform->_process_submission()
line 68 of /blocks/scheduledcontent/schedules.php: call to moodleform->__construct()

Notice: Undefined property: stdClass::$id0 in /var/www/t1.eduvidual.org/web/blocks/scheduledcontent/schedules.php on line 136

Notice: Undefined property: stdClass::$id0 in /var/www/t1.eduvidual.org/web/blocks/scheduledcontent/schedules.php on line 145
```

Inhalt im Dialogfenster does not work anymore (on t1?)

**Comment by rschrenk** [comment]
this is only a code style issue.
solved in df234d0f42e7357a5a7ff93b54700af1fc3d343a, will be part of next upgrade
